Cabal/Distribution/Compat/DList.hs

```diff
@@ -0,0 +1,36 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Distribution.Compat.DList
+-- Copyright   :  (c) Ben Gamari 2015-2019
```

Comment:
OK, let's merge, I can tackle the removal of `copyright` headers separately at some point in the future.